src/components/YouTube.tsx

```diff
@@ -27,7 +22,7 @@ const YouTube: React.FC<IProps> = ({ id, start = "0", title = "YouTube" }) => {
   const baseUrl = "https://www.youtube.com/embed/"
   const src = baseUrl + id + startQuery
   return (
-    <Figure>
+    <Box as="figure" display="block" marginY="1rem">
       <iframe
```

Comment:
I don't think we'd want to refactor the iframe to Chakra. LMK if you disagree

Reply:
No, thats ok 👍🏼